hledger-lib/Hledger/Data/Types.hs

```diff
@@ -214,6 +214,7 @@ instance Eq Posting where
 -- | The position of parse errors (eg), like parsec's SourcePos but generic.
 -- File name, 1-based line number and 1-based column number.
 data GenericSourcePos = GenericSourcePos FilePath Int Int
+                      | LedgerSourcePos FilePath (Int, Int) -- ^ range of lines in ledger journal
```

Comment:
Oh I see. I think "Journal" still works; in this context I think of hledger, ledger and even beancount files as "journal formats", but not time logs or CSV files.